microprot/scripts/calculate_Neff.py

```diff
@@ -61,10 +61,15 @@ def cluster_sequences(dm, cutoff):
     -------
     list of int
         flat cluster numbers to which sequences are assigned
+
+    Notes
+    -----
+    returns an empty list if the distance matrix is empty (e.g., there is only
+    one input sequence)
     """
     t = 1.0 - cutoff / 100.0
     return list(fcluster(linkage(dm.condensed_form()), t,
```

Comment:
I like this syntax and used it a lot in Perl, but I have the feeling it is not very pythonic. What about:
```
If dm.size <= 0:
	return []
return list(fcluster(linkage(dm.condensed_form()), t,criterion='distance'))
```